homework29/src/ba/bitcamp/homework29/HtmlCss/football.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+	<meta charset="utf-8">
+	<link rel="stylesheet" href="main.css">
+	<title>BARCELONA</title>
+
+</head>
+<header class="header">
+	
+	<img src="http://pre13.deviantart.net/e20c/th/pre/f/2014/304/3/c/fc_barcelona_header_for_facebook_and_twitter_by_zoooro-d84salt.png" style="
+    height: 150px;
+    width: 100%;
+">
+	
+	</header>
+	<br>
+	</br>
+
+<body>
+
+<div class="main">
+	<table id="navigation">
+	<tr>
+		<td><a href="file:///C:/Users/User/Desktop/football/gallery.html"> Gallery</a></td>
+		<td><a href="file:///C:/Users/User/Desktop/football/players.html"> Players</a></td>
+		<td><a href="file:///C:/Users/User/Desktop/football/history.html"> History</a></td>
+		<td><a href="file:///C:/Users/User/Desktop/football/press.html"> Press</a></td>
```

Comment:
Ovo su apsolutni linkovi. Sta to znaci? To znaci da je navedena puna lokacija fajla, koji postoji samo na racunaru na kojem se izvrsavla stanica - ne znaci da postoji i na drugim racunarima. Tako da npr. ovi linkovi kod mene ne rade. Potrebno je koristiti relativne linkove. To su linkovi cija lokacije pocinje od lokacije trenutnog fajla. Ako umjesto ovih apsolutnih linkova bili linkovi poput gallery.html, players.html, itd. onda bi takvi fajlovi bili ucitani iz trenutnog direktorija ovog fajla - sto bi se moglo otvoriti na svakom racunaru.